src/app/components/core/configuration-manager/view-config-item/view-config-item.component.ts

```diff
@@ -146,6 +148,49 @@ export class ViewConfigItemComponent implements OnInit, OnChanges {
     this.isWizardCall = true;
   }
 
+  public uploadScript(configItem) {
+    const fi = this.fileInput.nativeElement;
+    if (fi.files && fi.files[0]) {
+      const fileToUpload = fi.files[0];
+      const formData = new FormData();
+      formData.append('script', fileToUpload);
+      this.ngProgress.start();
+      this.configService.uploadFile(this.categoryConfiguration.key, configItem, formData)
+        .subscribe(() => {
+          this.ngProgress.done();
+          this.alertService.success('File uploaded Successfully');
+          this.getConfigItem(configItem);
```

Comment:
why do we need this?

Reply:
To get the content of latest uploaded file in `textarea`. But if there will be no `upload` button, then there is no need of this call `this.getConfigItem(configItem);`